package.json

```diff
@@ -111,7 +114,7 @@
     "mermaid": "^8.0.0-rc.8",
     "moment": "^2.20.1",
     "react-icons": "^2.2.5",
-    "react-jsonschema-form": "^1.2.0",
+    "react-jsonschema-form": "mozilla-services/react-jsonschema-form#174e136",
```

Comment:
Would it make sense to use `1.2.1` here, the newly released one ? It should contain commit `174e136`